port.js

```diff
@@ -0,0 +1,39 @@
+
+class Port {
+    label = 'unassigned';
+    layer = 'unassigned';
+    pos;
```

Comment:
Should this be set to a default coord?

Reply:
I created a default constructor for the coord and initialized it to that.